lib/wise_homex/models/statement_file.ex

```diff
@@ -0,0 +1,12 @@
+defmodule WiseHomex.StatementFile do
+  @moduledoc false
+
+  use WiseHomex.BaseModel
+
+  embedded_schema do
+    belongs_to :statement, WiseHomex.Statement
+
+    field :version, :integer
+    field :approval_date, :date
+  end
```

Comment:
This also `has_one` `external_resource`, which is where the presigned URL and description is stored.

Reply:
Yes, I noticed that, but there is no `ExternalResource` model in WiseHomex yet. So I thought I could wait with that until it's necessary (since it's related to many other models as well)

Reply:
Ah ok :+1: